### PR TITLE
Ya salik bil huda

### PR DESCRIPTION
**Pull request by thenotepadwriter**
Salaams!

It may still have some errors - I can correct them when I notice them isA. Maybe you prefer not to merge until it's checked- up to you. (I couldn't find a full recording to check).

Jzk!

### CODE REVIEW

**Review by adehad — APPROVED** [review]
Wa Salaams! Perfectly fine with me to adjust later.

From the Git perspective we've entered a merge conflict which is because it thinks there are changes on your branch that are also in the targeted branch. Let me know if you'll need any advice on that.

**Comment by adehad** [comment]
I just realised I have the poem from the Songs of Presence, so we can actually check with that too

**Comment by thenotepadwriter** [comment]
Assalamu alaikom!

Ah, that's great it's in songs of presence, and totally my bad that I didn't think to check there! If you have a PDF/photo, I'd be happy to cross check and update. 

Re: the conflict- yes, I saw... Was that because I added these commits on an existing/outdated fork instead of creating a new one? What's the best way to resolve that? (And if you're busy, I can also look it up and figure it out when I get a chance isA...)

Thanks!

**Comment by adehad** [comment]
The conflicts were because the `main` branch of this repository was different to the `main` branch on your repository (due to the previous contributions were from your `main` branch), so we can see how it says '6 commits ahead'
image

Ideally the `main` branch should only be behind, I know how to fix it from my laptop but not using the web interface to be honest. Maybe a way to fix it would be to delete the fork after we merged this and then re-fork it for the next contribution?

File: site/content/Qasidas/Ya-Salik-Bil-Huda [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Ya Salik Bil-Huda Tariq al-Bakri - O traveler on al-Bakri's guided way
+
+## Verses
+
```
Comment (adehad) [inline site/content/Qasidas/Ya-Salik-Bil-Huda:4]:
```suggestion
+++
maqam = ['Sika']
+++
# Ya Salik Bil-Huda Tariq al-Bakri - O traveler on al-Bakri's guided way

{{< qasida_meta >}}

## Verses

```

We can technically also do this now

**Pull request merged**